test/e2e/config/docker.yaml

```
@@ -367,12 +367,12 @@ variables:
   # allowing the same e2e config file to be re-used in different Prow jobs e.g. each one with a K8s version permutation.
   # The following Kubernetes versions should be the latest versions with already published kindest/node images.
   # This avoids building node images in the default case which improves the test duration significantly.
-  KUBERNETES_VERSION_MANAGEMENT: "v1.33.0"
-  KUBERNETES_VERSION: "v1.33.0"
-  KUBERNETES_VERSION_UPGRADE_FROM: "v1.32.2"
-  KUBERNETES_VERSION_UPGRADE_TO: "v1.33.0"
+  KUBERNETES_VERSION_MANAGEMENT: "v1.34.0-beta.0"
+  KUBERNETES_VERSION: "v1.34.0-beta.0"
+  KUBERNETES_VERSION_UPGRADE_FROM: "v1.33.3"
```

```suggestion
  KUBERNETES_VERSION_UPGRADE_FROM: "v1.33.1"
```
This version is in the mapper, also there is an image for it on DockerHub, i.e. we don't have to built it on every e2e test run

---

Got it, thanks!